packages/@guardian/source-foundations/src/typography/data.ts

```diff
@@ -17,6 +17,8 @@ const lineHeights = [1.15, 1.35, 1.5] as const;
 
 const fontWeights = [300, 400, 500, 700] as const;
 
+const underlineThickness = [2, 3, 4, 5, 6, 7] as const;
```

Comment:
Considering the number of comments required after all the implemenations of underlineThickness[x], would it be worth storing them in an object, with the pixel thickness in the key e.g. something like ```{
"thickness2px": 2,
"thickness3px": 3, 
... etc
}```? It might make it easier to tell at a glance the exact thickness that is being applied, and would remove the need for comments. I realise this isn't necessarily consistent with the rest of the codebase though...

Reply:
I did discuss this with @sndrs and he suggested something similar - we felt that maybe the whole structure of this file could be refactored in such a way that it removed the need for comments. 

Something along the lines of this may be simplest of all..

```js
const headlineUnderlineThickness = {
	xxxsmall: 3px,
	xxsmall: 4px,
	xsmall: 4px,
	small: 4px,
	medium: 5px,
	large: 6px,
	xlarge: 7px,
} as const;
```

For the sake of keeping things consistent while making these changes maybe it makes sense to get this merged, and then refactor the whole thing in this spirit in a separate PR?